### PR TITLE
URO-177: Align favicon to production KBase favicon.

### PR DESCRIPTION
**Pull request by dakotablair**
- [x] URO-177:
  - [x] The favicon in Europa is the same as the favicon on [kbase.us](http://kbase.us/).

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,21 +1,11 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "KBase",
```
Comment (dauglyon) [inline public/manifest.json:2]: KBase/ui?

File: src/app/App.test.tsx [hunk 2]
```diff
@@ -36,7 +39,9 @@ describe('The main UI for Europa...', () => {
   test('contains a working link to the Navigator', async () => {
     const { container } = render(
       <Provider store={createTestStore()}>
-        <App />
```
Comment (dauglyon) [inline src/app/App.test.tsx:39]: I think there's more in this PR than you intended

Comment (dakotablair) [reply src/app/App.test.tsx]: Now that the fallback code has been merged, I went ahead and rebased this branch to take it out of this PR.

File: public/manifest.json (hunk 1 above)
Comment (dakotablair) [reply public/manifest.json]: Updated `name` to be `kbase/ui` and removed `short_name` since `name` is less than 12 characters.

**Review by dauglyon — APPROVED** [review]

**Pull request merged**